shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -447,7 +447,6 @@ public Options getOptions() {
     executionHintsOpt = new Option(null, "execution-hints", true, "Execution hints map");
     scanServerOpt =
         new Option("cl", "consistency-level", true, "set consistency level (experimental)");
```

Comment:
Just going to re-add this blank line, because I think it's better here, to separate the option initializations and the arg names.

```suggestion
        new Option("cl", "consistency-level", true, "set consistency level (experimental)");

```